apps/nerves_hub_api/lib/nerves_hub_api_web/controllers/ca_certificate_controller.ex

```diff
@@ -27,16 +27,19 @@ defmodule NervesHubAPIWeb.CACertificateController do
          aki <- Certificate.get_aki(cert),
          ski <- Certificate.get_ski(cert),
          {not_before, not_after} <- Certificate.get_validity(cert),
-         params <- %{
-           serial: serial,
-           aki: aki,
-           ski: ski,
-           not_before: not_before,
-           not_after: not_after,
-           der: X509.Certificate.to_der(cert),
-           description: Map.get(params, "description")
-         },
-         {:ok, ca_certificate} <- Devices.create_ca_certificate(org, params) do
+         params <-
+           %{
+             serial: serial,
+             aki: aki,
+             ski: ski,
+             not_before: not_before,
+             not_after: not_after,
+             der: X509.Certificate.to_der(cert),
+             description: Map.get(params, "description")
+           }
+           |> maybe_merge_jitp(org, params),
```

Comment:
Does it mean we can create JITP entry from API ?

Reply:
This is explained in the commit message...